dracut/99journald-conf/00-journal-log-forwarding.conf

```diff
@@ -1,3 +1,13 @@
 [Journal]
+# For now we are using kmsg for multiplexing output to
+# multiple console devices during early boot. We need to also tell 
+# the kernel not to ratelimit kmsg during the initramfs.
```

Comment:
The ratelimiting isn't handled here, so perhaps drop that part?

Reply:
done